pims/tests/test_common.py

```diff
@@ -38,16 +38,14 @@ def _skip_if_no_ImageIO():
 
 
 def _skip_if_no_libtiff():
-    try:
-        import libtiff
-    except ImportError:
+    import pims.tiff_stack
```

Comment:
Sure, just saw other `import`s in functions that were not trying to catch `ImportError`s (e.g. a few lines above) and was doing this to be consistent. Would be totally fine doing it differently.